README.md

```diff
@@ -20,6 +20,9 @@ From source:
 git clone https://github.com/willthames/ansible-lint
 export PYTHONPATH=$PYTHONPATH:`pwd`/ansible-lint/lib
 export PATH=$PATH:`pwd`/ansible-lint/bin
+cd ansible-lint
+mkdir bin
+ln -s ../lib/ansiblelint/main/__init__.py bin/ansible-lint
```

Comment:
True, although if you're fundamentally averse to pip, you won't want to get virtualenv that way. (But you might be able to get it another way, e.g. Amazon Linux has a python27-virtualenv package.)

It probably depends what you're trying to do. If you really don't want to touch anything whatsoever on your system other than a git clone in your home directory (maybe you're using Ansible as a non-privileged user, who knows), a symlink is a way to run it out of a clone without touching anything else. I agree that a virtualenv is probably better for a lot of people; and, as @willthames suggested in #254, if you like package managers and don't like pip, you may be able to install it via yum or apt or whatnot.

I don't know how much @willthames wants to clutter the docs with alternatives; because there's *currently* instructions for running directly out of a clone, and at least two people were using it that way before, that seems like a reasonable thing to keep instructions for. But I don't personally mind if folks want to include virtualenv, yum, apt, etc, as additional options.